src/tic_tac_toe/input.clj

```diff
@@ -5,13 +5,20 @@
 
 (defn prompt-terminal [question]
    (println question)
-   (read-string(read-line)))
+   (read-line))
 
 (defn valid-selection [input board]
   (and (number? input) (validmove? input) (moveopen? board input)))
 
-(defn user-input-move [board ]
- (let [input (prompt-terminal "what is your next move ?")]
+(defn user-marker []
```

Comment:
If I enter anything besides "x" or "o" my input just defaults to "o". Is that how this incorrect input should be handled?